src/main/java/com/airlines/dao/impl/CrewMemberDaoImpl.java

```diff
@@ -0,0 +1,107 @@
+package com.airlines.dao.impl;
+
+import com.airlines.dao.CrewMemberDao;
+import com.airlines.dao.DbConnector;
+import com.airlines.exception.DaoOperationException;
+import com.airlines.model.Citizenship;
+import com.airlines.model.CrewMember;
+import com.airlines.model.Position;
+
+import java.sql.Connection;
+import java.sql.Date;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Objects;
+
+public class CrewMemberDaoImpl implements CrewMemberDao {
+
+    private final static String INSERT_CREW_MEMBER_SQL =
```

Comment:
SQL is structured query language, this constant does not contains language, but single sql query, so

INSERT_CREW_MEMBER_QUERY

would be better

Reply:
done in https://github.com/pielena/airlines-jdbc-app/commit/21a3eec6b8bb439efba0fe86562c5f31b291e47e